Makefile

```diff
@@ -20,9 +20,11 @@ API_TYPES_GENERATED:=$(API_TYPES_DIR)/$(API_ZZ_GENERATED).go
 API_GO_HEADER_FILE:=pkg/apis/header.go.txt
 
 # Container image-related variables
-DOCKERFILE=Dockerfile
-IMAGE_TAG=openshift/origin-cluster-node-tuning-operator
-IMAGE_REGISTRY=quay.io
+DOCKERFILE	?= Dockerfile.rhel7
```

Comment:
Why the changes in syntax and spacing?

Reply:
@sjug syntax back again, it was my VScode linter, and I didn't checked